schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT NOT NULL,
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL NOT NULL,
+    species VARCHAR(100),
```

Comment:
Great job @Htetaungkyaw71 🤝

- [ ] I would suggest you use the `ALTER TABLE animals ADD species varchar(100);` command to alter the already existing table instead of creating a new table with the species column.